platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch

```diff
@@ -248,10 +248,10 @@ index f1536e80e..4b214b22d 100644
  package com.google.android.vending.expansion.downloader.impl;
  
 -import com.android.vending.expansion.downloader.R;
-+// -- GODOT start --
++// -- BLAZIUM start --
 +//import com.android.vending.expansion.downloader.R;
```

Comment:
The lines 251-252 maybe from a different branch, please validate this is required for rebranding